examples/multi-container/ruby/example.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  annotations: 
+     run.googleapis.com/launch-stage: ALPHA
+  name: multicontainer-service
+spec:
+  template:
+    metadata:
+      annotations:
+        run.googleapis.com/execution-environment: gen1 #or gen2
+    spec:
+      containers:
+      - name: my-app
+        image: gcr.io/<YOUR_PROJECT_ID>/run-cloudsql:latest
+        ports:
+          - containerPort: 8080
+        env:
+          - name: DB_USER
+            value: <DB_USER>
+          - name: DB_PASS
+            value: <DB_PASS>
+          - name: DB_NAME
+            value: <DB_NAME>
+          - name: INSTANCE_HOST
+            value: "127.0.0.1"
+          - name: DB_PORT
+            value: "5432"
+      - name: cloud-sql-proxy
+        image: gcr.io/cloud-sql-connectors/cloud-sql-proxy:latest
+        args:
+             # If connecting from within a VPC network, you can use the
+             # following flag to have the proxy connect over private IP
+             # - "--private-ip"
+
+             - "--port=5432"
+             - "<INSTANCE_CONNECTION_NAME>"
```

Comment:
maybe add a comment here specifying format of instance connection name. `# instance connection name takes format "PROJECT:REGION:INSTANCE_NAME"`?

Reply:
✅